prusti-common/src/vir/ast/expr.rs

```diff
@@ -99,11 +107,18 @@ pub enum ContainerOpKind {
     // more to follow if required
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub enum FloatConst {
+    FloatConst64(u64),
+    FloatConst32(u32),
```

Comment:
What is the issue with storing `f64/32` directly? They would end up being the same IEEE representations, no?

Reply:
That is how the Rust compiler stores them.